DQM/SiPixelPhase1Heterogeneous/python/SiPixelPhase1HeterogenousDQM_FirstStep_cff.py

```diff
@@ -8,3 +8,41 @@
 
 
 monitorpixelSoASource = cms.Sequence(siPixelPhase1MonitorRecHitsSoA * siPixelPhase1MonitorTrackSoA * siPixelPhase1MonitorVertexSoA)
+
+
+#Define the sequence for GPU vs CPU validation
+#This should run:- individual monitor for the 2 collections + comparison module
+from DQM.SiPixelPhase1Heterogeneous.siPixelPhase1CompareTrackSoA_cfi import *
+from DQM.SiPixelPhase1Heterogeneous.siPixelPhase1CompareVertexSoA_cfi import *
+
+siPixelPhase1MonitorTrackSoACPU = siPixelPhase1MonitorTrackSoA.clone(
+  pixelTrackSrc = 'pixelTracksSoA@cpu',
```

Comment:
Just to make sure, the point is to always run these in the same job that produces `pixelTracksSoA` and `pixelVerticesSoA`, right?

Reply:
Yes, currently both RECO and DQM should run in the same job.

Once we have a "persistable" SoA solution, we may run this in different jobs (e.g. HLT and online DQM).